static/app/components/profiling/FlamegraphZoomView.tsx

```diff
@@ -1,11 +1,11 @@
-import * as React from 'react';
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
```

Comment:
Should we remove the default import for `React` completely while we're here?

Reply:
Nah, we use a fragment below in the render code, that's why I kept it. Kinda annoying :/